lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -76,7 +76,7 @@
 
     privatekey_passphrase:
         description:
-            - The passphrase for the I(privatekey_path).
+            - The passphrase for the I(privatekey_path). This may be required if the privatekey is password protected.
```

Comment:
```suggestion
            - The passphrase for the I(privatekey_path).
            - This may be required if the private key is password protected.
```

Reply:
I would also change the second sentence to `This is required if the ...` (i.e. `may be` → `is`).

Also, can you do the same change in `openssl_csr`?

Reply:
@felixfontein sure, in this PR?

Reply:
Yep, that's no problem.